system/prometheus-kubernetes/Chart.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v2
+description: Common Kubernetes Prometheus
+name: prometheus-kubernetes
+version: 0.0.1
+home: https://github.com/sapcc/helm-charts/tree/master/system/prometheus-kubernetes
+dependencies:
+  - name: prometheus-server
+    repository: https://charts.eu-de-2.cloud.sap
+    version: 7.2.7
```

```suggestion
    version: 7.2.8
```

+ new Chart.lock pls.